ODATA-1525 (#151)
oasis-tcs/odata-specs: odata-protocol/10 Context URL.md

nested `$select `or` $expand`, but MUST still be present, without a
suffix, if it is explicitly selected.

> There is no shortcut for representing the list of all navigation properties;
> `$expand=*` is treated as if the corresponding navigation properties were explicitly expanded.

Reviewer: ```suggestion
The context URL has no shortcut for representing the list of all navigation properties;
`$expand=*` is treated as if all navigation properties were explicitly expanded.
```
Is that what you mean?

Author: Thanks, that is much clearer
